assets/src/amp-block-editor.js

```diff
@@ -6,9 +6,15 @@ import { FormToggle, Notice } from '@wordpress/components';
 import { Fragment, RawHTML } from '@wordpress/element';
```

Comment:
This might not be the best approach. I renamed `assets/src/amp-block-editor-toggle.js` to `assets/src/amp-block-editor.js`, as I added the featured image notices to it. A more general name that doesn't have 'toggle' made more sense then.

But maybe there's a better file to put this logic. I wanted to reuse `getFeaturedImageNotice` and `getPrePublishNotice` from the `components/` modules.

Reply:
wfm